src/engine/controls/bpmcontrol.cpp

```diff
@@ -130,6 +130,15 @@ BpmControl::BpmControl(const QString& group,
             this, &BpmControl::slotTapFilter,
             Qt::DirectConnection);
 
+    m_pToggleBpmLock = std::make_unique<ControlPushButton>(
+            ConfigKey(group, "bpm_toggle_lock"), false);
+    m_pToggleBpmLock->setKbdRepeatable(true);
```

Comment:
Meeh, overlooked this one, too.
It's a toggle, makes no sense to allow repeat, might even cause issues if the mapped key (keyboard) is held down accidentally.

Will push a fix directly to main.